features/unit/v2algodclient_responses.feature

```diff
@@ -91,3 +91,11 @@ Feature: Algod REST API v2 Responses
     Examples:
       |jsonfiles           |directory                   |err|roundNum|
       |suggestedParams.json| v2algodclient_responsejsons||6222155|
+
```

Comment:
Add `@unit.dryrun` here, then I think we can merge this in

Reply:
done